src/utils/revocation.js

```diff
@@ -17,8 +38,97 @@ export function createRandomRegistryId() {
   return randomAsHex(RevRegIdByteSize);
 }
 
+/**
```

Comment:
i moved a few methods from utils/vc into this file as they can all be considered as vc revocation